src/containers/Bounty/index.js

```diff
@@ -229,7 +227,7 @@ class BountyComponent extends React.Component {
                     to={`/profile/${bounty.user.public_address}`}
                     addressTextColor="white"
                     size="small"
-                    border="true"
+                    border={true}
```

Comment:
you can just have the border prop in there... you don't need border={true}

Reply:
so `border=true`?

Reply:
no =